blazor/dropdown-list/placeholder-and-floatlabel.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: Placeholder/FloatLabel in Blazor DropDown List Component | Syncfusion
+description: Checkout and learn here all about Placeholder and FloatLabel in Syncfusion Blazor DropDown List component and more.
+platform: Blazor
+control: DropDown List
+documentation: ug
+---
+
+# Placeholder and Float Label in Dropdown List
+
+## Placeholder in the blazor dropdownlist component
+
+You can use [Placeholder](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_Placeholder) property to display small description of the expected value in the input. In the below sample demonstration, you have to set `Select a game` as `Placeholder` property value, which will set respective value to the `Placeholder` attribute of the input element in the DOM.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/placeholder-and-floatlabel/placeholder.razor %}
+
+{% endhighlight %}
+
+![Blazor DropdownList with placeholder](./images/placeholder-and-floatlabel/blazor_dropdown_placeholder.png)
+
+## Color of the placeholder text
+
+Use the following code to customize the text color of `Placeholder`.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/placeholder-and-floatlabel/placeholder-with-color.razor %}
+
+{% endhighlight %}
+
+![Blazor DropdownList with color placeholder](./images/placeholder-and-floatlabel/blazor_dropdown_placeholder-with-color.png)
+
+## Add mandatory indicator using placeholder
+
+Use the following code to add the mandatory indicator * to the float label element.

```

Comment:
Need to explain how we give "*" symbol in words. Since we achieve this in CSS content.